packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/private/helpers.ts

```diff
@@ -64,3 +64,25 @@ export function createLoggingOptions(scope: Construct, props: DestinationLogging
   }
   return undefined;
 }
+
+export function createBufferingHints(
+  interval?: cdk.Duration,
+  size?: cdk.Size,
+): firehose.CfnDeliveryStream.BufferingHintsProperty | undefined {
+  if (!interval && !size) {
+    return undefined;
+  }
+
+  const intervalInSeconds = interval?.toSeconds() ?? 300;
+  const sizeInMBs = size?.toMebibytes() ?? 5;
```

Comment:
I would move this variable definition closer to where it's first used (after the `if` in the following line).